design-documents/graph-ql/coverage/CartAddressOperations.graphqls

```diff
@@ -0,0 +1,84 @@
+type Query {
+    addBillingAddressToCart(input: AddBillingAddressToCartInput): AddBillingAddressToCartOutput
+    addShippingAddressToCart(input: AddShippingAddressToCartInput): AddShippingAddressToCartOutput
+}
+
+input AddBillingAddressToCartInput {
+    address_id: Int
+    address: CartAddressInput!
+}
+
+input AddShippingAddressToCartInput {
+    address_id: Int # Can be provided in one-page checkout and is required for multi-shipping checkout
+    address: CartAddressInput
+    shipping_method_code: String # Should be on address level for multi-shipping checkout
+}
+
+input CartAddressInput {
+    firstname: String!
+    lastname: String!
+    company: String
+    street: [String!]!
+    city: String!
+    region: String
+    postcode: String
+    country_code: String!
+    telephone: String!
+    fax: String
```

Comment:
do we really need fax anymore? I would get rid of it if we can do it without breaking things

Reply:
Done